test/DocumentFormat.OpenXml.Tests/Spreadsheet/CellValueTests.cs

```diff
@@ -74,6 +76,26 @@ public void CellDoubleTest(double num)
             Assert.Equal(num, result);
         }
 
+        [Fact]
+        public void CellDoubleCultureTest()
+        {
+            // Change current culture
+            CultureInfo culture, oldculture;
```

Comment:
can you create a helper for this? Something like this:

```
internal readonly struct CultureInfoTester:IDisposable
{
private readonly CultureInfo _old;

public CultureInfoTester(CultureInfo desired){
_old = Thread.CurrentThread.CurrentCulture;
Thread.CurrentThread.CurrentCulture = desired;
}

public void Dispose(){
Thread.CurrentThread.CurrentCulture = _old;
}
}
```

This can then be used to wrap the methods

Reply:
So something like this? I just pushed the helper.